docs/intro/clients.rst

```diff
@@ -0,0 +1,391 @@
+    anyhow = "1.0.63"
+
+  .. code-tab:: bash#Go
+
+    $ go get github.com/edgedb/edgedb-go
+
+  .. code-tab:: bash#.NET
+
+    $ dotnet add package EdgeDB.Net.Driver
+
+Copy and paste the following simple script. This script initializes a
+``Client`` instance. Clients manage an internal pool of connections to your
+database and provide a set of methods for executing queries.
+
+.. note::
+
+  Note that we aren't passing connection information (say, a connection
+  URL) when creating a client. The client libraries can detect that
+  they are inside a project directory and connect to the project-linked
+  instance automatically. (More on this later.)
+
+.. tabs::
+
+  .. code-tab:: typescript#Node.js
+
+    import {createClient} from 'edgedb';
+
+    const client = createClient();
+
+    client.querySingle(`select random()`).then((result) => {
+      console.log(result);
+    });
+
+
+  .. code-tab:: typescript#Deno
+
+    import {createClient} from 'https://deno.land/x/edgedb';
+
+    const client = createClient();
+
+    const result = await client.querySingle(`select random()`);
+    console.log(result);
+
+  .. code-tab:: python#Python
+
+    from edgedb import create_client
+
+    client = create_client()
+
+    result = client.query_single("select random()")
+    print(result)
+
+  .. code-tab:: rust#Rust
+
+    // src/main.rs
+    #[tokio::main]
+    async fn main() -> anyhow::Result<()> {
+        let conn = edgedb_tokio::create_client().await?;
+        let val = conn
+            .query_required_single::<f64, _>("select random()", &())
+            .await?;
+        println!("Result: {}", val);
+        Ok(())
+    }
+
+  .. code-tab:: go#Go
+
+    // hello.go
+    package main
+
+    import (
+      "context"
+      "fmt"
+      "log"
+
+      "github.com/edgedb/edgedb-go"
+    )
+
+    func main() {
+      ctx := context.Background()
+      client, err := edgedb.CreateClient(ctx, edgedb.Options{})
+      if err != nil {
+        log.Fatal(err)
+      }
+      defer client.Close()
+
+      var result float64
+      err = client.
+        QuerySingle(ctx, "select random();", &result)
+      if err != nil {
+        log.Fatal(err)
+      }
+
+      fmt.Println(result)
+    }
+
+  .. code-tab:: csharp#.NET
+
+    using EdgeDB;
+
+    var client = new EdgeDBClient();
+
+    var result = await client.QuerySingleAsync<double>("select random();");
+    Console.WriteLine(result);
+
+Finally, execute the file.
+
+.. tabs::
+
+  .. code-tab:: bash#Node.js
+
+    $ npx tsx index.ts
+
+  .. code-tab:: bash#Deno
+
+    $ deno run --allow-all --unstable index.deno.ts
+
+  .. code-tab:: bash#Python
+
+    $ python index.py
+
+  .. code-tab:: bash#Rust
+
+    $ cargo run
+
+  .. code-tab:: bash#Go
+
+    $ go run .
+
+  .. code-tab:: bash#.NET
+
+    $ dotnet run
+
+You should see a random number get printed to the console. This number was
+generated inside your EdgeDB instance using the built-in ``random()``
+function.
```

Comment:
```suggestion
function from Python.
```

These are various languages, developers outside of the Python language will be confused.

Reply:
This is referring to the `random` function that's built into EdgeQL: https://www.edgedb.com/docs/stdlib/numbers#function::std::random 

I added a link to avoid confusion 👍